tests/acceptance/features/apiSharingNg/propfindShares.feature

```diff
@@ -43,3 +43,33 @@ Feature: propfind a shares
       | oc:permissions   | SR               |
       | oc:size          | 8                |
       | d:getcontenttype | text/plain       |
+
+  @issue-4421
+  Scenario: sharee PROPFIND a shares when multiple users share folder with same name
```

Comment:
Is it possible to put this and above scenario in example table?

Reply:
are they necessary (related to the bug report)?

Reply:
> are they necessary (related to the bug report)?

I don't think so. I will refactor with an example table.